server/controllers/userControllers.js

```diff
@@ -42,7 +42,8 @@ class UserController {
                     othername: userResponse[0].othername,
                     email: userResponse[0].email,
                     phone: userResponse[0].phone,
-                    passportUrl: userResponse[0].passportUrl,
+                    passportUrl: userResponse[0].passporturl,
+                    isAdmin: userResponse[0].isadmin,
                 };
                 const token = jwt.sign(data, process.env.SECRET_KEY, { expiresIn: '2d' });
```

Comment:
What is contained in your data object, you use to create your jwt token

Reply:
The user's details including the name, email, phone .... and isAdmin status